src/db/db_test.rs

```diff
@@ -5,13 +5,16 @@ mod test {
     #[cfg(test)]
     mod single_txn_simple_test {
         use std::sync::Arc;
+        use tempfile::tempdir;
 
         use crate::db::db::{Timestamp, DB};
 
         #[tokio::test]
         async fn two_writes_with_different_keys() {
-            let db = Arc::new(DB::new_random_path(Timestamp::new(10)));
-
+            let db = Arc::new(DB::new_cleaned(
```

Comment:
I was wondering if a helper method like `DB::new_tempdir(...)` would make it easier to create tests in the future.

Reply:
Added helper function under `db_test.rs` as `create_temp_dir`. I think its better to limit the scope of such helper functions to tests instead of exposing a public function from `db.rs` just for doing the test setup.

Reply:
Agreed. In the future we can probably create a `TestDB` struct and we can do `TestDB::create(...)`.